lib/index.js

```diff
@@ -197,9 +207,8 @@ class PluginPassportOAuth {
           return this.context.accessors.execute(new this.context.constructors.Request(request.original, req, {refresh: 'wait_for'}))
             .then(res => Promise.resolve({kuid: res.result._id, message: null}));
         }
-        return Promise.resolve({kuid: null, message: `Could not login with strategy "${profile.provider}"`});
-      })
-      .catch(err => Promise.reject(err));
+        return Promise.reject(new NotFoundError());
```

Comment:
Error messages should not be empty.

Nitpickings: 
* you can throw instead of returning a promise rejection
* you can invert the `if` condition just above, and throw in it, instead of having a big chunk of code in that condition